person.rb

```diff
@@ -23,10 +23,10 @@ def can_use_services?
   def of_age?
     @age >= 18
   end
-end
 
-def add_rentals(_person, date)
-  Rental.new(date, book, self)
-end
+  def add_rentals(_person, date)
+    Rental.new(date, book, self)
```

Comment:
- Hey, you seem to have missed this part of the changes requested by the previous reviewer. You are supposed to update the setter method to include the :rentals as he mentioned giving the updated code. please, kindly remove this line and make this update to your setter method. `attr_accessor :name, :age, :rentals`